test/python/qobj/test_pulse_converter.py

```diff
@@ -36,7 +36,7 @@ class TestInstructionToQobjConverter(QiskitTestCase):
     def test_deprecated_drive_instruction(self):
         """Test converted qobj from PulseInstruction."""
         converter = InstructionToQobjConverter(PulseQobjInstruction, meas_level=2)
-        command = SamplePulse(np.arange(0, 0.01), name='linear')
+        command = Waveform(np.arange(0, 0.01), name='linear')
```

Comment:
Let's use this test to test the deprecated SamplePulse instruction:
```suggestion
        with self.assertWarns(DeprecationWarning):
            command = SamplePulse(np.arange(0, 0.01), name='linear')
```